### PR TITLE
Update domains-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
added a note about implications when you change the default domain.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6251

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/domains-faq.yml [hunk 1]
```diff
@@ -144,6 +144,9 @@ sections:
           
           ::: moniker-end
           
+          > [!NOTE]
+          > Changing the default domain does not impact the user logon or access to application or services. Only If you change a UPN, any saved links to the user's OneDrive (such as desktop shortcuts or browser favorites) will no longer work and will need to be updated.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml]:
```suggestion
          > Changing the default domain does not impact the user logon or access to applications or services. Only if you change a UPN will any saved links to the user's OneDrive (such as desktop shortcuts or browser favorites) no longer work and will need to be updated.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum  @scotvidican  Please merge this PR. Thanks

**Pull request merged**